app/src/main/java/com/jin/jjinweather/layer/domain/usecase/GetWeatherUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.jin.jjinweather.layer.domain.usecase
+
+import com.jin.jjinweather.layer.domain.model.weather.Weather
+import com.jin.jjinweather.layer.domain.repository.WeatherRepository
+
+class GetWeatherUseCase(private val repository: WeatherRepository) {
+    suspend operator fun invoke(): Weather {
+        return repository.loadWeather()
+    }
```

Comment:
```suggestion
    suspend operator fun invoke(): Weather = repository.loadWeather()
```
간단한 한 줄 짜리 function body는 expression body로 바꿔쓰는 것이 kotlin convention입니다.
https://kotlinlang.org/docs/coding-conventions.html#expression-bodies

Reply:
앗! 참고하겠습니다!